pkg/compose/plugins.go

```diff
@@ -79,50 +97,55 @@ func (s *composeService) runPlugin(ctx context.Context, project *types.Project,
 	variables := types.Mapping{}
 
 	pw := progress.ContextWriter(ctx)
-	pw.Event(progress.CreatingEvent(service.Name))
+	var action string
+	switch command {
+	case "up":
+		pw.Event(progress.CreatingEvent(service.Name))
+		action = "create"
+	case "down":
+		pw.Event(progress.RemovingEvent(service.Name))
+		action = "remove"
+	}
```

Comment:
Should this switch also have a default? Looks like if `command` is not `up` or `down`, then the switch at the end of the function returns an error (but after a bunch of work is done). Maybe that's intentional, but I just noticed it.

If it's intentional, it would be good to add a `default:` with a comment that explains that it is intentional (but will produce an error further down).

Reply:
Yep, I added the `default` to the wrong switch 🤦